reference/supervisor.md

```diff
@@ -767,9 +745,37 @@ indicates the success of the save operation. Be aware that this function can
 overwrite silently existing files, so that the corresponding data may be lost.
 
 > **Note** [C++, Java, Python, Matlab]:
-In the other APIs, the `Robot.saveWorld` function can be called without
+In the other APIs, the `Robot.worldSave` function can be called without
 argument. In this case, a simple save operation is performed.
 
+The `wb_supervisor_world_reload` function sends a request to the simulator
+process, asking it to reload the current world immediately. As a result of
+reloading the current world, the supervisor process and all the robot processes
+are terminated and restarted. You may wish to save some data in a file from your
+supervisor program in order to reload it when the supervisor controller
+restarts.
+
+---
+
+**Name**
+
+**wb\_supervisor\_simulation\_reset** - *reset the simulation*
+
+{[C++](cpp-api.md#cpp_supervisor)}, {[Java](java-api.md#java_supervisor)}, {[Python](python-api.md#python_supervisor)}, {[Matlab](matlab-api.md#matlab_supervisor)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_simulation_reset();
+```
+
+**Description**
+
+The `wb_supervisor_simulation_reset` function sends a request to the simulator process, asking it to reset the simulation immediately.
+The reset process is explained in detail in the [User Guide](https://www.cyberbotics.com/doc/guide/the-user-interface#file-menu).
```

Comment:
Is this link still correct if a specific version is displayed?
For example when opening the online doc for a specific release?

Reply:
It will work but the `version` will be removed.
As long as #327 is not fixed, I don't see any other way to reference another book.

Reply:
Yes, it will link to the wrong documentation and the possibly wrong explanation of the reset functionality.
This is bad, and we should find a solution.
For me it can be fixed in another issues, but it is important to fix this soon.

Reply:
👍 
I did set the milestone of the issue to R2018b so that we make sure it is fixed before this become public.